src/test/java/org/neo4j/spark/SparkConnectorSuiteIT.java

```diff
@@ -0,0 +1,11 @@
+package org.neo4j.spark;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+
+})
```

Comment:
i guess that you missed this?
```suggestion
@Suite.SuiteClasses({DataSourceReaderTypesTSE.class})
```

Reply:
Of course I did 😓